packages/checkout-extension/src/ExtensionService.ts

```diff
@@ -104,4 +59,37 @@ export class ExtensionService {
 
         return Boolean(extension);
     }
+
+    private registerHandlers(extension: Extension): void {
+        const handlerProps = {
+            checkoutService: this.checkoutService,
+            dispatch: this.dispatch,
+            extension,
+        };
+
+        if (!this.handlers[extension.id]) {
+            this.handlers[extension.id] = [];
+        }
+
+        const isCommandHandler = <T extends keyof ExtensionCommandMap>(
```

Comment:
Minor comment, does it make sense to move this as a private method outside this function?